www/source/index.html.slim

```diff
@@ -32,188 +26,195 @@ section.sub-hero
   .row
     .columns.medium-12
       .sub-hero--content
-        h2.sub-hero--heading Why Habitat?
+        h2.sub-hero--heading What does Habitat do?
         .row
-          .sub-hero--blurb.columns.medium-4
-            img.sub-hero--icon[src="/images/icons/subhero-manage.svg"
-              onerror="this.src='/icons/subhero-manage.png'" alt=""]
-            h4 Freedom to run anywhere
+          .sub-hero--blurb.columns.medium-3
+            .sub-hero--blurb-title
+              img.sub-hero--icon[src="/images/icons/subhero-build.svg"
+                onerror="this.src='/icons/subhero-build.png'" alt=""]
+              h4 Build
             p
-              | Run greenfield and legacy applications anywhere, from bare
-                metal to containers and PaaS. Habitat puts the focus on the app.
+              | Define your app dependencies, startup scripts, and configuration
+                settings, then use the Habitat Studio to confidently build
+                clean, lean artifacts.
 
-          .sub-hero--blurb.columns.medium-4
-            img.sub-hero--icon[src="/images/icons/subhero-results.svg"
-              onerror="this.src='/icons/subhero-results.png'" alt=""]
-            h4 Support for production environments
+          .sub-hero--blurb.columns.medium-3.medium-offset-1
+            .sub-hero--blurb-title
+              img.sub-hero--icon[src="/images/icons/subhero-deploy.svg"
+                onerror="this.src='/icons/subhero-deploy.png'" alt=""]
+              h4 Deploy
             p
-              | Make your app production ready with monitoring, runtime
-                configuration, rolling deployment, and more.  Put an end to
-                "last-mile" configuration nightmares.
+              | Ship your Habitat build artifact as-is or export it into other
```

Comment:
I think if your build artifact is singular, the export format should be singular too, so it should be "export it into another format, like a container"